packages/react-core/src/demos/DataListDemo.md

```diff
@@ -418,3 +424,899 @@ class ExpandableDataList extends React.Component {
+          {filterDropdownItems}
+        </Select>
+      </ToolbarFilter>
+    );
+  }
+
+  render() {
+    const {
+      isUpperToolbarDropdownOpen,
+      isLowerToolbarDropdownOpen,
+      isUpperToolbarKebabDropdownOpen,
+      isLowerToolbarKebabDropdownOpen,
+      isCardKebabDropdownOpen,
+      splitButtonDropdownIsOpen,
+      activeItem,
+      filters,
+      res,
+      checked,
+      selectedItems,
+      areAllSelected,
+      isChecked,
+      page,
+      perPage
+    } = this.state;
+
+    const toolbarKebabDropdownItems = [
+      <OverflowMenuDropdownItem itemId={0} key="link">
+        Link
+      </OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem itemId={1} key="action" component="button">
+        Action
+      </OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem itemId={2} key="disabled link" isDisabled>
+        Disabled Link
+      </OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem itemId={3} key="disabled action" isDisabled component="button">
+        Disabled Action
+      </OverflowMenuDropdownItem>,
+      <Divider key="separator" />,
+      <OverflowMenuDropdownItem itemId={5} key="separated link">
+        Separated Link
+      </OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem itemId={6} key="separated action" component="button">
+        Separated Action
+      </OverflowMenuDropdownItem>
+    ];
+
+    const toolbarItems = (
+      <React.Fragment>
+        <ToolbarItem variant="bulk-select">{this.buildSelectDropdown()}</ToolbarItem>
+        <ToolbarItem breakpoint="xl">{this.buildFilterDropdown()}</ToolbarItem>
+        <ToolbarItem variant="overflow-menu">
+          <OverflowMenu breakpoint="md">
+            <OverflowMenuItem>
+              <Button variant="primary">Create a project</Button>
+            </OverflowMenuItem>
+            <OverflowMenuControl hasAdditionalOptions>
+              <Dropdown
+                onSelect={this.onToolbarKebabDropdownSelect}
+                toggle={(toggleRef) => (
+                  <MenuToggle
+                    ref={toggleRef}
+                    aria-label="Toolbar kebab overflow menu"
+                    variant="plain"
+                    onClick={this.onToolbarKebabDropdownToggle}
+                    isExpanded={isLowerToolbarKebabDropdownOpen}
+                  >
+                    <EllipsisVIcon />
+                  </MenuToggle>
+                )}
+                isOpen={isLowerToolbarKebabDropdownOpen}
+                onOpenChange={(isOpen) => this.setState({ isLowerToolbarKebabDropdownOpen: isOpen })}
+              >
+                <DropdownList>{toolbarKebabDropdownItems}</DropdownList>
+              </Dropdown>
+            </OverflowMenuControl>
+          </OverflowMenu>
+        </ToolbarItem>
+        <ToolbarItem variant="pagination" align={{ default: 'alignRight' }}>
+          {this.renderPagination()}
+        </ToolbarItem>
+      </React.Fragment>
+    );
+
+    const filtered =
```

Comment:
This `filtered` variable, and the `icons` variable defined on line 970 are not used anywhere. you can remove them.